levels/test_level.tscn

```diff
@@ -1,11 +1,12 @@
-[gd_scene load_steps=8 format=3 uid="uid://dwt6ner4kappn"]
+[gd_scene load_steps=9 format=3 uid="uid://dwt6ner4kappn"]
 
 [ext_resource type="PackedScene" uid="uid://cabj5tagd7116" path="res://environments/cave_1.tscn" id="1_jnsrf"]
 [ext_resource type="Script" path="res://levels/landing_pad.gd" id="2_cweo1"]
 [ext_resource type="Material" uid="uid://b5c507hor1smc" path="res://materials/glow_green.tres" id="2_l1ue5"]
 [ext_resource type="PackedScene" uid="uid://brs3why3rugsi" path="res://scenes/player/player.tscn" id="3_lfap1"]
 [ext_resource type="PackedScene" uid="uid://bwwd8ds1by6v4" path="res://scenes/trigger_pad.tscn" id="4_av7qf"]
 [ext_resource type="PackedScene" uid="uid://7e87dwxuo4u2" path="res://scenes/sliding_door.tscn" id="5_nut2n"]
+[ext_resource type="PackedScene" uid="uid://dke233bj6pxqx" path="res://pointer.tscn" id="7_e7hyl"]
```

Comment:
This reference should also be removed, since the `pointer.tscn` no longer exists.
```suggestion
```